internal/pubsub/mempubsub/mem.go

```diff
@@ -37,8 +52,16 @@ type topic struct {
 
 // OpenTopic establishes a new topic.
 // Open subscribers for the topic before publishing.
-func OpenTopic() driver.Topic {
-	return &topic{}
+func (b *Broker) OpenTopic(ctx context.Context, name string) *pubsub.Topic {
```

Comment:
Since we decided to not have a `Client` in the "real" impls, and instead have top-level `OpenXXX` functions, WDYT about doing that here as well? I.e., something like:

```
func OpenTopic(b *Broker, name string) *pubsub.Topic
```

Reply:
Done.